### PR TITLE
Fix live log 500 error and some code cleanup

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -781,8 +781,22 @@ def set_job_status
   def live_build_log
     required_parameters :arch, :repository
 
-    @project = Project.get_by_name(params[:project])
-    @package = Package.get_by_project_and_name(params[:project], params[:package], use_source: false, follow_project_links: true)
+    if Project.exists_by_name(params[:project])
```
Comment (hennevogel) [inline src/api/app/controllers/webui/package_controller.rb]: Why are we not using the general before filters (`set_project`, `require_package`) here?

Comment (bgeuken) [inline src/api/app/controllers/webui/package_controller.rb]: I don't know.

Comment (bgeuken) [inline src/api/app/controllers/webui/package_controller.rb]: Maybe because we need to cover remote projects too

**Comment by bgeuken** [comment]
@adrianschroeter done squashing the 2 changes

**Comment by bgeuken** [comment]
I rebased to solve the merge conflict. That made adrians comment disappear. Here it is: https://github.com/bgeuken/open-build-service/commit/86fc9c9a7d9fde4047b299b18572d6c4f69689b1

**Pull request merged**